kite-data/kite-data-mapreduce/src/main/java/org/kitesdk/data/mapreduce/DatasetKeyOutputFormat.java

```diff
@@ -334,11 +335,20 @@ public void close(TaskAttemptContext taskAttemptContext) {
     }
   }
 
-  static class NullOutputCommitter extends OutputCommitter {
+  static class NullOutputCommitter<E> extends OutputCommitter {
     @Override
     public void setupJob(JobContext jobContext) { }
 
     @Override
+    public void commitJob(JobContext jobContext) throws IOException {
+      View<E> targetView = load(jobContext);
+
+      if(targetView instanceof ReadiableView) {
+        ((ReadiableView)targetView).signalReady();
```

Comment:
What is your reasoning behind automatically signalling that the output is ready? I don't want to accumulate a lot of unused files in the signal directory. Does it make sense to add a configuration option to signal the output when it is completed?